setup.go

```diff
@@ -175,8 +175,21 @@ func (o *OverflowBuilder) StartResult() *OverflowState {
 	if o.InputResolver != nil {
 		overflow.InputResolver = *o.InputResolver
 	} else {
-		overflow.InputResolver = func(name string) (string, error) {
-			return overflow.QualifiedIdentifierFromSnakeCase(name)
+		overflow.InputResolver = func(name string, resolveType underflow.ResolveType) (string, error) {
+			if resolveType == underflow.Identifier {
+				return overflow.QualifiedIdentifierFromSnakeCase(name)
+			}
+
+			adr, err2 := hexToAddress(name)
+			if err2 == nil {
+				return adr.String(), nil
+			}
+
+			address, err2 := overflow.FlowAddressE(name)
+			if err2 != nil {
+				return "", errors.Wrapf(err2, "could not parse %s into an address", name)
+			}
+			return address.HexWithPrefix(), nil
```

Comment:
should this be provided by overflow as a default or just exported func?

Reply:
this should be the default, other implementations are really only there for testing.